Files/View Models/SettingsViewModel.cs

```diff
@@ -324,6 +324,8 @@ public bool PinRecycleBinToSideBar
                                 App.sideBarItems.Remove(item);
                             }
                         }
+
+                        App.JumpList.RemoveFolder(RecycleBinPath);
```

Comment:
I don't think it's necessary to remove recycle bin from the recent items list. The only thing disabling recycle bin should do right now, is to unpin it from the sidebar.